website/pages/en/index.js

```diff
@@ -293,10 +293,10 @@ const UsersSection = ({ language }) => {
             </a>
             <div style={{ marginLeft: ".7em", width: "260px" }}>
               <p>
-                Regularly used by more than <strong>70%</strong> of respondents
-                to State of JS 2020
+                Regularly used by more than <strong>83%</strong> of question
+                respondents to State of JS 2021
```

Comment:
"question respondents **to** State of JS 2021" - is this correct English? A rewording seems to be needed. Any ideas?